great_expectations/expectations/metrics/util.py

```diff
@@ -666,73 +662,101 @@ def get_dbms_compatible_column_names(
     Args:
         column_names: Single string-valued column name or list of string-valued column names
         batch_columns_list: Properly typed column names (output of "table.columns" metric)
-        execution_engine: "ExecutionEngine" (here, of interest is whether or not "SqlAlchemyExecutionEngine" is used)
         error_message_template: String template to output error message if any column cannot be found in "Batch" object.
 
     Returns:
         Single property-typed column name object or list of property-typed column name objects (depending on input).
     """
-    verify_column_names_exist(
-        column_names=column_names,
-        batch_columns_list=batch_columns_list,
-        error_message_template=error_message_template,
+    normalized_typed_batch_columns_mappings: List[
+        Tuple[str, str | sqlalchemy.sql.quoted_name]
```

Comment:
nit - we don't need `List` and `Tuple` we can just use `list` and `tuple`

Reply:
@cdkini Without `List` and `Tuple`, how can we specify what the internal types are?..  Thanks.